styles/index.css

```diff
@@ -12,7 +12,181 @@ body {
   background-color: #e5e5e5;
 }
 
-/* header */
+/* main */
+#main {
+  display: flex;
+  flex-direction: column;
+}
+
+/* CLASS TAKE PRECEDE */
+.nav-buttons {
+  margin: 12px 0;
+  display: flex;
+  flex-direction: row;
+}
+
+.nav-buttons li {
+  height: 40px;
+  width: auto;
+  border: 1px solid #8993a4;
+  text-decoration-style: none;
+  margin-right: 12px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+}
+
+.l-f-skills a {
+  text-decoration: none;
+  font-family: Inter, fantasy;
+  font-size: 22px;
+  font-weight: 400;
+  text-align: center;
+  color: #fff;
+  padding-bottom: 32px;
+}
+
+.nav-buttons li a {
+  text-decoration: none;
+  padding: 12px 10px;
+  height: 40px;
+  width: auto;
+  font-family: Inter, fantasy;
+  font-size: 15px;
+  font-weight: 500;
+  text-align: center;
+  color: #091e42;
+}
+
+/* general button */
+.btn-type-1 {
+  height: 48px;
+  width: fit-content;
+  padding: 12px 12px;
+  font-family: Inter, fantasy;
+  font-size: 17px;
+  font-weight: 700;
+  line-height: 24px;
+  text-align: center;
+  color: #fff;
+  background-color: #ff6b00;
+  outline: none;
+  border: 1px solid transparent;
+}
+
+.btn-type-1:hover {
+  border: 1px solid #f55800;
+  background-color: #fff5e1;
+  color: #f55800;
+}
+
+.btn-type-1:active {
+  color: #f4f1f0;
+  background-color: #e05304;
+  border: 1px solid transparent;
+}
+
+.btn-type-1:disabled {
+  background-color: #5b5957;
+  border: 1px solid transparent;
+}
+
+/* class works-box */
+.works-box {
```

Comment:
- [ ] As demanded by the project requirements, kindly use flexbox to layout contents of each work card. Flexbox makes building responsive applications easier.